tests/security_features_test.go

```diff
@@ -188,6 +190,10 @@ var _ = Describe("[Serial][sig-compute]SecurityFeatures", Serial, decorators.Sig
 		Context("With selinuxLauncherType defined as virt_launcher.process", func() {
 
 			It("[test_id:4298]qemu process type is virt_launcher.process, when selinuxLauncherType is virt_launcher.process", func() {
+				if flags.DisableCustomSELinuxPolicy {
```

Comment:
I this we need a new label, see https://github.com/kubevirt/kubevirt/blob/main/tests/decorators/decorators.go#L5

Reply:
I gave it a shot, force pushed, PTAL.